src/controllers/bookController.ts

```diff
@@ -1,23 +1,53 @@
 import { Router, Request, Response } from 'express';
+import { Connection, Request as RequestStatement } from 'tedious';
+import { executeStatement } from './connectionConfig';
+import Book from '../bookClass';
 
 class BookController {
     router: Router;
 
     constructor() {
         this.router = Router();
-        this.router.get('/:id', this.getBook.bind(this));
+        this.router.get('/getBook/:id', this.getBook.bind(this));
+        this.router.get('/getAllBooks', this.getAllBooks.bind(this));
```

Comment:
This is absolutely fine, and I wouldn't suggest that you change it for this exercise. But, for your interest, it would be more conventional to just serve this at `/books`.

`GET /books` has sufficient semantic meaning to make it clear what we want to do.

This is very nitpicky and a bit subjective though, so don't worry about it for now.